test/ng/filter/orderBySpec.js

```diff
@@ -157,6 +183,38 @@ describe('Filter: orderBy', function() {
     });
 
 
+    it('should support a custom comparator', function() {
```

Comment:
I would go as far as calling it "nerdy" :stuck_out_tongue: 

I added a bunch of other tests, so it's not so necessary any more - I just kept it around because I liked it. I'm fine removing it if you think it's too confusing.